Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Link/Link.cpp

```diff
@@ -135,6 +135,24 @@ namespace AzToolsFramework
             return areOverridesPresent;
         }
 
+        PrefabDomConstReference Link::FindOverridePatch(
+            AZ::Dom::Path path,
+            AZ::Dom::PrefixTreeTraversalFlags prefixTreeTraversalFlags)
+        {
+            PrefabDomConstReference overridePatch = {};
+            auto visitorFn = [&overridePatch](AZ::Dom::Path, const PrefabOverrideMetadata& overrideData)
+            {
+                overridePatch = overrideData.m_patch;
+
+                // We just need to get one override at the provided path.
+                // Return false here so that we don't keep looking for all patches at the path.
+                return false;
```

Comment:
This will make sure that the function stops visiting other nodes in the tree if it encounters at least 1 patch. This will work for something like 'AreOverridesPresent' . But for something like 'FindOverridePatches', this should probably do something like GetLinkPatches() where it'll fetch all overrides at a given path.

Reply:
If there is an "add" or "remove" override present, won't there just be that one patch anyway (other patches get merged into the one)?

Reply:
I'll change to GetLinkPatches and later check number of patches anyway to address your other comment...